todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -91,34 +108,31 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         FloatingActionButton fab =
                 (FloatingActionButton) getActivity().findViewById(R.id.fab_edit_task);
 
-        fab.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                mPresenter.editTask();
-            }
-        });
+        // RxBindings
+        mEditTaskClick = RxView.clicks(fab);
+        mCheckedChanges = RxCompoundButton
+                .checkedChanges(mDetailCompleteStatus)
+                .doOnNext(checked -> Log.i("TEST", "Pre-UpdatingFilter " + checked))
+                .filter(checked -> !mUpdating)
```

Comment:
My main concern is about this kind of logic code in the View, since you can't unit test this.